notebooks/testing_suite.ipynb

```diff
@@ -224,18 +208,18 @@
     }
    ],
    "source": [
-    "train_map_at_k, train_ndcg_at_k = get_metrices_for_given_dataset(train_df, config)\n",
-    "valid_map_at_k, valid_ndcg_at_k = get_metrices_for_given_dataset(validation_df, config)\n",
-    "test_map_at_k, test_ndcg_at_k = get_metrices_for_given_dataset(test_df, config)\n",
-    "popular_test_map_at_k, popular_test_ndcg_at_k = get_metrices_for_given_dataset(popular_test_df, config)\n",
-    "unpopular_test_map_at_k, unpopular_test_ndcg_at_k = get_metrices_for_given_dataset(unpopular_test_df, config)\n",
+    "train_recall_at_k, train_map_at_k, train_ndcg_at_k = get_metrices_for_given_dataset(train_df, config)\n",
```

Comment:
Z tego co widzę w testing_framework.py, to get_metrices_for_given_dataset ma inną sygnaturę...
`get_metrices_for_given_dataset(config, hashtag_data, row, recommendation_function)`
Jak to powinno być rzeczywiście odpalane?